wls-auth-service/src/main/resources/db/migrations/h2/V12_0__addAdminPermissionsDeleteWahltermindaten.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO Permission
+VALUES ('00000000-0000-0012-0000-000000000001', 'Admin_BUSINESSACTION_DeleteWahltermindaten');
+INSERT INTO Secauthorities_Secpermissions
+VALUES ('00000000-0000-0000-0001-000000000001', '00000000-0000-0012-0000-000000000001');
```

Comment:
wenn ich es richtig verstanden habe, müsste hier für die admin authorities eine 2 stehen oder?
```suggestion
VALUES ('00000000-0000-0000-0002-000000000001', '00000000-0000-0012-0000-000000000001');
```

Reply:
Nein - die values in Tabelle `Secauthorities_Secpermissions` gehen in die Spalten `user_oid` und `authority_oid`.

Die `user_oid` ist für unseren User "wls_all" immer der Foreign_KEY "00000000-0000-0000-0001-000000000001".
Die `authority_oid` ist der FOREIGN_KEY zur Tabelle `Permission` und wird darüber definiert.